buildtest/utils/command.py

```diff
@@ -1,7 +1,69 @@
+import os
 import locale
 import subprocess
 import shlex
 import shutil
+import tempfile
```

Comment:
This was written by me, and shouldn't appear like this. It suggests that you didn't pull the merged commits from my contribution. It's not really okay to take credit for my work, so this needs to be fixed.

Reply:
I am not sure what happened i do see your git commit **Fix/command out err (#289)** in my commit history https://github.com/shahzebsiddiqui/buildtest-framework/commits/update_shell_logic 
The only other explanation could be that when i copy-paste to fix this i applied black so it's showing a diff.
I am bit afraid to do a merge from devel now